<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>差分包合并 For 某动漫游戏  |  Copyright by RainKavik</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/pic/RoundCorner.ico</normaloff>:/pic/RoundCorner.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始合并！</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>511</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>481</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>完整客户端路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="path50_textbox">
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="path50_button">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>差分包路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="path51_textbox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="path51_button">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
